Documentation/ExtensionArchitecture/FileStructure/ExtTablesStaticAdtSql.rst

```diff
@@ -9,28 +9,22 @@
 Static SQL tables and their data.
 
 If the extension requires static data you can dump it into an SQL file
-by this name. Example for dumping mysql data from bash (being in the
-extension directory):
+by this name. Example for dumping MySQL/MariaDB data from shell (executed in the
+extension's root directory):
 
 .. code-block:: shell
 
-   mysqldump --add-drop-table \
-               --password=[password] [database name] \
-               [tablename]  > ./ext_tables_static+adt.sql
+   mysqldump --user=[user] --password [database name] \
+             [tablename] > ./ext_tables_static+adt.sql
 
-:code:`--add-drop-table` will make sure to include a DROP TABLE
-statement so any data is inserted in a fresh table.
+Note that only :sql:`INSERT INTO` statements are allowed. If the contents of the
+SQL file change, the table is truncated and the new data is inserted.
```

Comment:
We could use the chance to specify when exactly the file gets parsed; this suggests a change to the file would trigger re-loading the static data, but that is not the case. It gets a bit complicated though; the "reload action" in the EM for example is only available when the file contents already changed. Don't know how to explain that in a short way.

An idea:

```
Note that only :sql:`INSERT INTO` statements are allowed. The file is interpreted
whenever the corresponding extension's setup routines get called: Upon first time installation,
command task execution of :bash:`bin/typo3 extension:setup` or via the :guilabel:`Admin Tools > Extensions` interface and the :guilabel:"Reload extension data" action. The static data is then only re-evaluated, if the file has different contents than on the last execution.
```

We may need crosslinks to the extension manager and this specific "reload extension data" feature at some point? Couldn't easily find it right now. We havehttps://docs.typo3.org/m/typo3/tutorial-getting-started/main/en-us/Extensions/LegacyManagement.html#installing-an-extension-using-the-extension-manager which doesn't adress the "reload" action. Not sure if a "Getting started" tutorial is the right place for it, too.

Reply:
Adjusted the paragraph.